pi-apps-terminal-bash-edition.sh

```diff
@@ -209,7 +209,7 @@ while [[ "$1" != "" ]]; do
 				IFS=$'\n'
 				for folder in "$DIRECTORY"/apps/*; do
 					if [[ "$app" == "$(basename "${folder,,}")" ]]; then
-						app="$folder"
+						app="${folder##*/}"
```

Comment:
```suggestion
						app="$(basename "$folder")"
```
I prefer using `basename` as its more clear what is happening.